protobuf/scalapb/scalapb.proto

```diff
@@ -2,7 +2,10 @@ syntax = "proto2";
 
 package scalapb;
 
+option csharp_namespace = "ScalaPb.Options";
```

Comment:
protoc accepts this file without this change, so this modification may break compatibility for users. Lets remove this line.